src/Compilers/CSharp/Portable/Symbols/PublicModel/NamespaceOrTypeSymbol.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols.PublicModel
+{
+    internal abstract class NamespaceOrTypeSymbol : Symbol, INamespaceOrTypeSymbol
+    {
+        internal abstract Symbols.NamespaceOrTypeSymbol UnderlyingNamespaceOrTypeSymbol { get; }
+
+        ImmutableArray<ISymbol> INamespaceOrTypeSymbol.GetMembers()
+        {
+            return UnderlyingNamespaceOrTypeSymbol.GetMembers().SelectAsArray(m => m.GetPublicSymbol());
```

Comment:
>.SelectAsArray(m => m.GetPublicSymbol()) [](start = 63, length = 40)

Gould just call `GetPublicSymbols()`

Reply:
And below.

---
In reply to: [338829341](https://github.com/dotnet/roslyn/pull/39498#discussion_r338829341) [](ancestors = 338829341)